cmd/smith/app/bundle_controller.go

```diff
@@ -226,7 +227,10 @@ func (c *BundleControllerConstructor) New(config *ctrl.Config, cctx *ctrl.Contex
 		Broadcaster: broadcaster,
 		Recorder:    recorder,
 	}
-	cntrlr.Prepare(crdInf, resourceInfs)
+	err = cntrlr.Prepare(crdInf, resourceInfs)
```

Comment:
:thinking: this should have been a warning before...

Reply:
It wasn't returning an error before this PR.